Makefile

```diff
@@ -141,14 +145,16 @@ $(PROTO_MODEL): data/unicharset data/radical-stroke.txt
 	  --output_dir data \
 	  --lang $(MODEL_NAME)
 
+#      19 [1,48,0,1Ct3,3,16Mp3,3Lfys64Lfx96Lrx96Lfx512O1c1]
```

Comment:
Why this comment?

Reply:
It's a network specifications which was used rather often (19 times) for the Tesseract models. The comment should not be there, so I'll remove it from the PR.

Reply:
The comment is removed now.